openapi/spec/dm.yaml

```diff
@@ -297,6 +297,11 @@ components:
           type: string
           example: ""
           description: "Content of the private key file in X509 format"
+        cert_allowed_cn:
+          type: array
+          description: "Common Name of ssl certs"
```

Comment:
SSL

Reply:
fixed in https://github.com/pingcap/dm/pull/2055/commits/dae765d7920c5b76f2b1873e1f6a70fca818dbfb